test/e2e/framework.go

```diff
@@ -116,7 +116,7 @@ const (
 	nameSuffixLength int = 8
 
 	agnhostImage        = "k8s.gcr.io/e2e-test-images/agnhost:2.29"
-	busyboxImage        = "projects.registry.vmware.com/library/busybox"
+	busyboxImage        = "projects.registry.vmware.com/tanzu_adv_eng/busybox"
```

Comment:
you could push a busybox image to `projects.registry.vmware.com/antrea` instead?

Reply:
Updated. Thanks @tnqn for pushing image.